test/protocol/models/message_test.dart

```diff
@@ -1,31 +1,27 @@
-import 'dart:convert';
-
 import 'package:tbdex/src/protocol/models/message.dart';
-import 'package:tbdex/src/protocol/models/order.dart';
-import 'package:tbdex/src/protocol/models/rfq.dart';
 import 'package:test/test.dart';
 
 import '../../test_data.dart';
 
 void main() async {
   await TestData.initializeDids();
 
-  group('Message Test', () {
-    test('can parse a list of messages', () async {
-      final rfq = TestData.getRfq();
-      await rfq.sign(TestData.aliceDid);
-      final order = TestData.getOrder();
-      await order.sign(TestData.pfiDid);
+  group('MessageTest', () {
```

Comment:
For test group names, I would typically leave off the `Test` part. So this would be `group('Message', ` since it's testing the `Message` class

Reply:
ahh right, will fix that up!